agents/acu/acu_agent.py

```diff
@@ -407,25 +423,43 @@ def start_monitor(self, session, params=None):
-                for statkey in self.data['status'][category].keys():
-                    if type(self.data['status'][category][statkey]) == float:
-                        if self.data['status'][category][statkey] == float('nan'):
-                            influx_status[statkey + '_influx'] = 0.0
-                        else:
-                            influx_status[statkey + '_influx'] = self.data['status'][category][statkey]
-                    elif type(self.data['status'][category][statkey]) == str:
-                        if self.data['status'][category][statkey] in ['None', 'True', 'False']:
-                            influx_status[statkey + '_influx'] = tfn_key[self.data['status'][category][statkey]]
-                        else:
-                            influx_status[statkey + '_influx'] = mode_key[self.data['status'][category][statkey]]
-                    elif type(self.data['status'][category][statkey]) == int:
-                        if statkey in ['Year', 'Free_upload_positions']:
-                            influx_status[statkey + '_influx'] = float(self.data['status'][category][statkey])
-                        else:
-                            influx_status[statkey + '_influx'] = int(self.data['status'][category][statkey])
-                    else:
-                        print(statkey)
-
+                if category != 'commands':
+                    for statkey, statval in self.data['status'][category].items():
+                        if type(statval) == float:
+                            influx_status[statkey + '_influx'] = statval#self.data['status'][category][statkey]
+                        elif type(statval) == str:
+                            if statval == 'None':
+                                influx_status[statkey + '_influx'] = float('nan')
+                            elif statval in ['True', 'False']:
+                                influx_status[statkey + '_influx'] = tfn_key[statval]#self.data['status'][category][statkey]]
+                            else:
+                                influx_status[statkey + '_influx'] = mode_key[statval]#self.data['status'][category][statkey]]
+                        elif type(statval) == int:
+                            if statkey in ['Year', 'Free_upload_positions']:
+                                influx_status[statkey + '_influx'] = float(statval)#self.data['status'][category][statkey])
+                            else:
+                                influx_status[statkey + '_influx'] = int(statval)#self.data['status'][category][statkey])
+                       # else:
+                       #     print(statkey)
+                elif category == 'commands':
+                    if str(self.data['status']['commands']['Azimuth_commanded_position']) != 'nan':
+                        acucommand_az = {'timestamp': self.data['status']['summary']['ctime'],
+                                         'block_name': 'ACU_commanded_positions',
```

Comment:
Thanks for making this one feed.  Technically, you should give each field set a different block_name.  They are just for  internal bookkeeping so you can just call them 'az', 'el', 'boresight'.

Since this is an a feed for influx instead of the aggregator, using the same value ("ACU_commanded_positions") for all three sets of fields might not cause trouble... but it's good to be consistent.